main.js

```diff
@@ -0,0 +1,69 @@
+//2D array representing three pegs on the board
+let startStack = ["5","4","3","2","1"];
+
+var board = {
+  boardState: [
+  ["5", "4", "3", "2", "1"],
+  [],
+  []],
+
+  moveDisc: function (startPeg, endPeg) {
+    
+    let startArr = this.boardState[startPeg - 1];
+    let endArr = this.boardState[endPeg - 1];
+
+    if(endArr.length === 0 ||  startArr[startArr.length - 1] < endArr[endArr.length - 1]) {
+    endArr.push(startArr.pop());
+    console.log('That move was successful. The board is now: ');
+    this.boardState.map(x => console.log(`--- ${x.join(' ')}`));
+    } else {
+      console.log('You can not move a larger disc on top of a smaller one, board is still: ');
+      this.boardState.map(x => console.log(`--- ${x.join(' ')}`));
+      }
+      board.checkWinner(endArr);
+  },
+
+  checkWinner: function (arr) {

```

Comment:
I would consider calling this helper something different as it does not necessarily encompass all the logic necessary to check the winner as it could be called with a different array.